src/components/GraphPopupMenu/GraphPopupMenu.js

```diff
@@ -50,6 +50,43 @@ const connectionPropertiesColors = {
     },
 }
 
+/**
```

Comment:
These edits are preventable by rebasing your branch based on dev. Tip for the next time :)

Reply:
Thanks for the tip! Apparently my rebase didn't work properly haha, but I'll make sure that the rebase is successful next time.